.github/actions/run-airbyte-ci/action.yml

```diff
@@ -149,6 +152,7 @@ runs:
         SLACK_WEBHOOK: ${{ inputs.slack_webhook_url }}
         SPEC_CACHE_BUCKET_NAME: ${{ inputs.spec_cache_bucket_name }}
         SPEC_CACHE_GCS_CREDENTIALS: ${{ inputs.spec_cache_gcs_credentials }}
+        CODEFLASH_API_KEY: ${{ inputs.codeflash_api_key }}
```

Comment:
@alafanechere am I missing something obvious here? After moving from a custom GHA to airbyte-ci, the workflow is failing because it can't access the `CODEFLASH_API_KEY`.

Reply:
I think you're missing one last change: the thing that would make `airbyte-ci` set this env var on the docker container running the poe task.
It has to be defined in `[tool.airbyte-ci]` in the `pyproject.toml` file. I'll add a suggestion comment below.